NOAA-OWP/cahaba
# [1pt] PR: Renaming of benchmark data and associated variables

The benchmark inundation rasters were named "depth" instead of "extent", and NWS and USGS domain shapefiles were named "extent" instead of "domain". This branch addresses those issues in the `test_cases` directory and in `run_test_case.py` as well as `run_test_case_calibration.py`, although the latter will likely be deprecated.

### Testing
To test, you can run:
```
python foss_fim/tools/run_test_case.py -r fim_3_0_4_4_ms_c_agg/12090301/ -b dev-rename-pr-nws -t 12090301_nws
```
and then run:
```
python foss_fim/tools/run_test_case.py -r fim_3_0_4_4_ms_c_agg/12090301/ -b dev-rename-pr-ble -t 12090301_ble
```

And confirm no Traceback errors and good-looking outputs.
